client/transport/tlsconfig_clone_go16.go

```diff
@@ -0,0 +1,13 @@
+// +build go1.6
+// +build !go1.7
```

Comment:
The line means `go1.6 && !go1.7`.
If we split the line it would become |go1.6 || !go1.7`
https://golang.org/pkg/go/build/

And even on Go 1.7, the `go1.6` tag is true, that's why we need |go1.7`